lang/uk/user.php

```diff
@@ -221,13 +221,13 @@
     'public-info'                          => 'Публічна інформація',
     'recent-achievements'                  => 'Останні досягнення',
     'recent-followers'                     => 'Останні послідовники',
-    'registration-date'                    => 'Дата Реєстрації',
+    'registration-date'                    => 'Дата реєстрації',
     'report'                               => 'Доповідь',
     'request-help'                         => 'Керуйте спільним доступом до певної статистики, пов язаної з запитом, та інформацією з групами, яким дозволено доступ до вашого профілю. Ці налаштування перевизначаються, якщо ви не дозволяєте будь-яким групам отримувати доступ до запитаної статистики та інформації, або якщо ви <strong>йдете на приват</strong>',
     'request'                              => 'Запит',
     'requested'                            => 'Запитано',
     'requests'                             => 'Запити',
-    'request-comments'                     => 'Запит Коментарі зроблено',
+    'request-comments'                     => 'Запит коментарів зроблено',
```

Comment:
I'm pretty sure this translation is incorrect. The string used to be used for the profile statistic of how many comments a user has made on torrent requests. Google translate tells me the new translation means "The request for comments has been made" which isn't related.

Reply:
It appears the updated translation is treating "request" as a verb instead of a noun. To give some context, an example of the string being used would be:

```
Torrent comments made: 4
Article comments made: 6
Request comments made: 8
Playlist comments made: 10
```

Can you confirm your translation provides this meaning? When using google translate, it returns "The comment request is made" which makes me think it's referring to an action being performed instead of the count of how many actions were performed.